financial-connections/src/main/java/com/stripe/android/financialconnections/exception/FinancialConnectionsError.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.financialconnections.exception
```

Comment:
is this used anywhere?

Reply:
Yes! It's not new, it was just on the wrong file and moved it to its own - https://github.com/stripe/stripe-android/pull/6794/files#diff-0d57241060a1753b1c4a26d5c27909e101a54013645f142e528cade0844320a2L15